app/modules/individuals/models.py

```diff
@@ -167,6 +167,21 @@ def get_custom_fields(self):
     def get_taxonomy_guid(self):
         return self.get_edm_data_field('taxonomy')
 
+    def get_taxonomy_names(self):
+        taxonomy_guid = self.get_edm_data_field('taxonomy')
+        taxonomy_names = []
+        # Taxonomy guid is optional in the response from EDM
+        if taxonomy_guid:
+            from app.modules.site_settings.models import SiteSetting
+
+            site_species = SiteSetting.get_value('site.species')
+            for species in site_species:
+                if site_species[species]['id'] == taxonomy_guid:
```

Comment:
this is kinda handy; wonder if it might serve us well in `app/utils.py` or something.  `taxonomy_guid` -> `site_species` dict.  just a thought.

Reply:
Interesting idea. I would use my usual filter on that one : If any of us find ourselves doing the same thing again, yup make a util.